onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -33,7 +33,10 @@ class TensorrtLogger : public nvinfer1::ILogger {
                "%Y-%m-%d %H:%M:%S",
                std::gmtime(&rawtime));
       const char* sevstr = (severity == Severity::kINTERNAL_ERROR ? "    BUG" : severity == Severity::kERROR ? "  ERROR" : severity == Severity::kWARNING ? "WARNING" : severity == Severity::kINFO ? "   INFO" : "UNKNOWN");
-      LOGS_DEFAULT(WARNING) << "[" << buf << " " << sevstr << "] " << msg;
+      if(severity <= Severity::kERROR)
+          LOGS_DEFAULT(ERROR) << "[" << buf << " " << sevstr << "] " << msg;
```

Comment:
Just curious, where was the default logger initialized?

Reply:
It wasn't, the 'OutputIsEnabled' stub function would always return 'false' so nothing would be logged in these cases. It was due to be fixed...